stratify/_vinterp.pyx

```diff
@@ -527,8 +529,8 @@ cdef class _Interpolation(object):
 
     cdef public np.dtype _target_dtype
     cdef int rising
-    cpdef public z_target, orig_shape, axis, _zp_reshaped, _fp_reshaped
-    cpdef public _result_working_shape, result_shape
+    cdef public z_target, orig_shape, axis, _zp_reshaped, _fp_reshaped
+    cdef public _result_working_shape, result_shape
```

Comment:
Changed these from `cpdef` to `cdef` to avoid `cython` future warnings, but functionally a `nop` change.